src/app/projects/create-project/create-project.controller.js

```diff
@@ -731,7 +734,7 @@ class CreateProjectCtrl {
 
       // type selected
       if (this.importProjectData.project.type) {
-        name = this.importProjectData.project.type;
+        name = this.importProjectData.project.type.replace(' ', '');
```

Comment:
I don't know if it's wanted but replace method only replace the first occurence, not all occurence like in java